src/backend/BSXML.pm

```diff
@@ -734,6 +734,7 @@ our $worker = [
 	'hostarch',
 	'ip',
 	'port',
+        'reposerver',
```

Comment:
Oh my. We already use 'reposerver' for masterdispatched jobs, so we need to use some other name here. Sorry.